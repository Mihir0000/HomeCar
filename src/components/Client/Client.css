.client_section {
    width: 100%;
    float: left;
    padding-bottom: 90px;
}

.layout_padding {
    padding-top: 90px;
    padding-bottom: 0px;
}


#my_slider .carousel-control-prev:hover,
#my_slider .carousel-control-next:hover,
#my_slider .carousel-control-prev:focus,
#my_slider .carousel-control-next:focus {
    background-color: #19e9d9;
    color: #ffffff;
}

#my_slider a.carousel-control-next {
    position: absolute;
    left: 490px;
    top: 70%;
    background-color: #19e9d9;
    color: #ffffff;
}

#my_slider a.carousel-control-prev {
    position: absolute;
    left: 490px;
    top: 84%;
    background-color: #252525;
}

#my_slider .carousel-control-prev,
#my_slider .carousel-control-next {
    width: 65px;
    height: 65px;
    background-color: #252525;
    opacity: 1;
    font-size: 30px;
    color: #ffffff;
}

.client_taital {
    width: 100%;
    float: left;
    font-size: 40px;
    text-align: center;
    color: #111010;
    font-weight: bold;
}

.client_text {
    width: 100%;
    float: left;
    font-size: 16px;
    color: #444443;
    text-align: center;
    margin: 0px;
}

.client_section_2 {
    width: 100%;
    float: left;
    padding-top: 60px;
}

.client_left {
    width: 40%;
    float: left;
}

.client_right {
    width: 60%;
    float: left;
    padding-top: 100px;
}

.client_img {
    width: 80%;
    margin: 0 auto;
    text-align: center;
    display: block;
}

.distracted_text {
    width: 100%;
    float: left;
    font-size: 40px;
    color: #080808;
    font-weight: bold;
    text-transform: uppercase;
}

.lorem_text {
    width: 100%;
    float: left;
    font-size: 16px;
    color: #444443;
    margin: 0px;
}

.quote_icon {
    width: 100%;
    float: left;
    padding-top: 20px;
}