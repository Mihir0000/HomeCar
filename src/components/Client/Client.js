import React from 'react';
import { Card, Container } from 'react-bootstrap';
import './Client.css';

const Client = () => {
    return (
        <Container fluid className="client_section layout_padding">
            <Container
                id="my_slider"
                className="carousel slide"
                data-ride="carousel"
            >
                <Container className="carousel-inner">
                    <Container className="carousel-item active">
                        <Card className="bg-transparent border-0">
                            <Card.Title className="client_taital fs-1">
                                What People Say
                            </Card.Title>
                            <Card.Text className="client_text">
                                It is a long established fact that a reader will
                                be distracted{' '}
                            </Card.Text>
                            <Card.Body className="client_section_2">
                                <Card.Text className="client_left">
                                    <Card.Img
                                        src="static/img/client-img.png"
                                        className="client_img p-3"
                                        alt="client-img"
                                    />
                                </Card.Text>
                                <Card.Title className="client_right">
                                    <Card.Subtitle className="distracted_text fs-2">
                                        Distracted by
                                    </Card.Subtitle>
                                    <Card.Text className="lorem_text">
                                        It is a long established fact that a
                                        reader will be distracted by the
                                        readable content of a page when looking
                                        at its layout. The point of using Lorem
                                        Ipsum is that it has a more-or-less
                                        normal distribution of letters
                                    </Card.Text>
                                    <Card.Text className="quote_icon">
                                        <Card.Img
                                            style={{ width: '8%' }}
                                            src="static/img/quote-icon.png"
                                            alt="img"
                                        />
                                    </Card.Text>
                                </Card.Title>
                            </Card.Body>
                        </Card>
                    </Container>
                    {/* <div className="carousel-item">
                        <div className="container">
                            <h1 className="client_taital">What People Say</h1>
                            <p className="client_text">
                                It is a long established fact that a reader will
                                be distracted{' '}
                            </p>
                            <div className="client_section_2">
                                <div className="client_left">
                                    <div>
                                        <img
                                            src="static/img/client-img.png"
                                            className="client_img"
                                            alt="client_img"
                                        />
                                    </div>
                                </div>
                                <div className="client_right">
                                    <h3 className="distracted_text">
                                        Distracted by
                                    </h3>
                                    <p className="lorem_text">
                                        It is a long established fact that a
                                        reader will be distracted by the
                                        readable content of a page when looking
                                        at its layout. The point of using Lorem
                                        Ipsum is that it has a more-or-less
                                        normal distribution of letters
                                    </p>
                                    <div className="quote_icon">
                                        <img
                                            src="static/img/quote-icon.png"
                                            alt="quote"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div className="carousel-item">
                        <div className="container">
                            <h1 className="client_taital">What People Say</h1>
                            <p className="client_text">
                                It is a long established fact that a reader will
                                be distracted{' '}
                            </p>
                            <div className="client_section_2">
                                <div className="client_left">
                                    <div>
                                        <img
                                            src="static/img/client-img.png"
                                            className="client_img"
                                            alt="client_img"
                                        />
                                    </div>
                                </div>
                                <div className="client_right">
                                    <h3 className="distracted_text">
                                        Distracted by
                                    </h3>
                                    <p className="lorem_text">
                                        It is a long established fact that a
                                        reader will be distracted by the
                                        readable content of a page when looking
                                        at its layout. The point of using Lorem
                                        Ipsum is that it has a more-or-less
                                        normal distribution of letters
                                    </p>
                                    <div className="quote_icon">
                                        <img
                                            src="static/img/quote-icon.png"
                                            alt="quote"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div> */}
                </Container>
                <Card.Link
                    className="carousel-control-prev"
                    href="#my_slider"
                    role="button"
                    data-slide="prev"
                >
                    <i
                        className="fa fa-long-arrow-left"
                        style={{ fontSize: '24px', paddingTop: '4px' }}
                    ></i>
                </Card.Link>
                <Card.Link
                    className="carousel-control-next"
                    style={{ marginLeft: '-1px' }}
                    href="#my_slider"
                    role="button"
                    data-slide="next"
                >
                    <i
                        className="fa fa-long-arrow-right"
                        style={{ fontSize: '24px', paddingTop: '4px' }}
                    ></i>
                </Card.Link>
            </Container>
        </Container>
    );
};

export default Client;
