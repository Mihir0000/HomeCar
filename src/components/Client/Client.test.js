import React from 'react';
import { render } from '@testing-library/react';
import ReactDOM from 'react-dom';

import renderer from 'react-test-renderer';
import Client from './Client';


test('snapshot testing --client', () => {
    const component = renderer.create(<Client />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

// crash test --client
it('render without crashing --client', () => {
    const div = document.createElement('div');
    render(<Client />, div);
    ReactDOM.unmountComponentAtNode(div);
});