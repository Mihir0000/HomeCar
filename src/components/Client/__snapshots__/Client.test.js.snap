// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing --client 1`] = `
<div
  className="client_section layout_padding container-fluid"
>
  <div
    className="carousel slide container"
    data-ride="carousel"
    id="my_slider"
  >
    <div
      className="carousel-inner container"
    >
      <div
        className="carousel-item active container"
      >
        <div
          className="bg-transparent border-0 card"
        >
          <div
            className="client_taital fs-1 card-title h5"
          >
            What People Say
          </div>
          <p
            className="client_text card-text"
          >
            It is a long established fact that a reader will be distracted
             
          </p>
          <div
            className="client_section_2 card-body"
          >
            <p
              className="client_left card-text"
            >
              <img
                alt="client-img"
                className="card-img client_img p-3"
                src="static/img/client-img.png"
              />
            </p>
            <div
              className="client_right card-title h5"
            >
              <div
                className="distracted_text fs-2 card-subtitle h6"
              >
                Distracted by
              </div>
              <p
                className="lorem_text card-text"
              >
                It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters
              </p>
              <p
                className="quote_icon card-text"
              >
                <img
                  alt="img"
                  className="card-img"
                  src="static/img/quote-icon.png"
                  style={
                    Object {
                      "width": "8%",
                    }
                  }
                />
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev card-link"
      data-slide="prev"
      href="#my_slider"
      role="button"
    >
      <i
        className="fa fa-long-arrow-left"
        style={
          Object {
            "fontSize": "24px",
            "paddingTop": "4px",
          }
        }
      />
    </a>
    <a
      className="carousel-control-next card-link"
      data-slide="next"
      href="#my_slider"
      role="button"
      style={
        Object {
          "marginLeft": "-1px",
        }
      }
    >
      <i
        className="fa fa-long-arrow-right"
        style={
          Object {
            "fontSize": "24px",
            "paddingTop": "4px",
          }
        }
      />
    </a>
  </div>
</div>
`;
